api/http.go

```diff
@@ -78,6 +78,12 @@ func NewCatalystAPIRouter(cli config.Cli, vodEngine *pipeline.Coordinator, bal b
 	for _, path := range [...]string{"/asset/hls/:playbackID/*file", "/webrtc/:playbackID"} {
 		router.OPTIONS(path, playback)
 	}
+	image := middleware.LogAndMetrics(metrics.Metrics.ImageRequestDurationSec)(
```

Comment:
Is it intended to be used directly from the public API or via Studio? Isn't it a risk that someone will access this endpoint directly without any authentication?